akka-actor/src/main/scala/akka/util/ConstantFun.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.stream.impl
+package akka.util
```

Comment:
I was thinking we perhaps should still just copy it and mark as deprecated.
The reason is that Akka HTTP did use those (!), so if we remove them, and someone upgrades akka but not Akka HTTP they would break.

I propose to keep it deprecated for one or two HTTP releases so we can move to touching the new one in akka.util.